app/scripts/lib/auth-errors.js

```diff
@@ -187,6 +187,10 @@ function (_, Errors) {
     INVALID_EMAIL: {
       errno: 1023,
       message: t('Valid email required')
+    },
+    FORCE_AUTH_EMAIL_REQUIRED: {
```

Comment:
This error message is [already being used](https://github.com/mozilla/fxa-content-server/blob/950700373bdb370469bbf2affc2851f12996dbda/app/scripts/views/force_auth.js#L39). I moved it here so it can be logged with an `errno`